test/Microsoft.AspNet.Mvc.FunctionalTests/Compiler/Resources/MvcTagHelpersWebSite.MvcTagHelper_Home.Link.html

```diff
@@ -7,19 +7,19 @@
     <title>Link</title>
 
     <!-- Plain link tag -->
-    <link href="/site.css" rel="stylesheet" />
+    <link href="/site.css" rel="stylesheet" title="&lt;the title>" />
 
     <!-- Globbed link tag with existing file -->
-    <link rel="stylesheet" href="/site.css" />
+    <link rel="stylesheet" title="&lt;the title>" href="/site.css" />
 
     <!-- Globbed link tag with existing file and exclude -->
-    <link rel="stylesheet" href="/site.css" /><link rel="stylesheet" href="/sub/site2.css" />
+    <link rel="stylesheet" title="&quot;the&quot; title" href="/site.css" /><link rel="stylesheet" title="&quot;the&quot; title" href="/sub/site2.css" />
 
     <!-- Globbed link tag missing include -->
     <link rel="stylesheet" />
 
     <!-- Globbed link tag missing include but with static href -->
-    <link href="/site.css" rel="stylesheet" />
+    <link rel="stylesheet" href="/site.css" />
```

Comment:
reordering happens because `href` is now bound.  I'll revert changes like this when I fix aspnet/Razor#225, which is mostly about moving bound attributes to the end.